test/test_torch.py

```diff
@@ -19024,6 +19024,46 @@ def make_index_negative(sequence, idx):
         np_fn = partial(np.moveaxis, source=(), destination=())
         self.compare_with_numpy(torch_fn, np_fn, x, device=None, dtype=None)
 
+    @dtypes(torch.int64, torch.float, torch.complex128)
+    def test_swapdim_invalid(self, device, dtype):
```

Comment:
For both of these tests use "transpose" in the names and start by iterating over the functions:

```
for fn in (torch.transpose, torch.swapdims, torch.swapaxes):
  ...
```